pkg/acquisition/modules/appsec/appsec.go

```diff
@@ -31,6 +32,8 @@ const (
 )
 
 var DefaultAuthCacheDuration = (1 * time.Minute)
+var negativeAllowlistCacheDuration = (5 * time.Minute)
+var positiveAllowlistCacheDuration = (5 * time.Minute)
```

Comment:
looks like a lot to me, but it depends if SE gets a `force_pull` via PAPI when the allowlist is updated ?